### DISCUSSION TITLE
Option to sort by ascending/descending

### DISCUSSION THREAD

**Discussion by SevaOnOss** [ppy/osu#21993] [Ideas] [answered]
Main reason I'm suggesting this is because lazer puts new beatmaps to the top when sorting by date instead of bottom like stable does. Most people are used to new maps being at the bottom (and it feels more natural) so atleast an option to change it would be nice

**Comment by peppy** [comment, marked as answer]
Yep, song select will get more sorting/grouping in the future. Please wait to see what's coming.

**Comment by StarpageIDK** [comment]
can't wait it to come to lazer, a lot of times Im getting confused by this lol
